client/driver/rkt.go

```diff
@@ -731,6 +951,36 @@ func (d *RktDriver) Open(ctx *ExecContext, handleID string) (DriverHandle, error
 	return h, nil
 }
 
+func (d *RktDriver) fetchImage(img lastditch.RuntimeImage, fetchArgs []string) (string, error) {
+	var version string
```

Comment:
Can we set `version` to `latest` by default and use that if there's no version specified instead of erroring out?